edit/completion/compl_getopt.go

```diff
@@ -91,6 +92,22 @@ func complGetopt(fm *eval.Frame, elemsv, optsv, argsv interface{}) {
 		if s, ok := getStringField("arg-desc"); ok {
 			argdesc[opt] = s
 		}
+
+		getCallableField := func(k string) (eval.Callable, bool) {
```

Comment:
Can you move the definition to just under `getStringField`?

Reply:
I also put all three functions (`getCallableField`,`getCallableField` and `getBoolField`) together, hope to make it more clear.